cis/system/hsp-cis-1.1.13-admin-conf-block.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorHostPolicy
+metadata:
+  name: hsp-cis-1.1.13-admin-conf-block
+spec:
+  tags: ["CIS", "1.1.13","K8", "V1.20"]
```

Comment:
Keep the same tag throughout the repo, kindly use `k8s` instead of `K8`

Reply:
Tags are changed.